packages/gatsby/src/utils/webpack.config.js

```diff
@@ -276,10 +292,14 @@ module.exports = async (
         // prettier-ignore
         configRules = configRules.concat([
           {
-            ...rules.css(),
-            use: [loaders.null()],
+            oneOf: [
+              rules.cssModules(),
```

Comment:
This could be improved by omitting the MiniCssExtractPlugin.loader like with the SASS and LESS loaders.

Reply:
merging this now but I'll add this as a TODO over on my PR